components/mixins/experience-event/experienceevent-environment-details.schema.json

```diff
@@ -30,6 +33,11 @@
           "title": "Place context",
           "$ref": "https://ns.adobe.com/xdm/context/placecontext",
           "description": "The transient circumstances related to the observation. Examples include locale specific information such as weather, local time, traffic, day of the week, workday vs. holiday, and working hours."
+        },
+        "xdm:platform": {
```

Comment:
All the properties under this mixin are nested in one of the following top level objects:
device
environment
placeContext

The new property "platform" added here will be at the root. Can this be added to any of the data types referred under the above top level objects ?

Reply:
environment could be an option. Is there a way we could have rachel give her comments on these as well. That would be great..

Reply:
@mjasrotia @prabhum2 
Let's move this to a V2 change and just approve the meta:tags for now?